src/login/logind-action.c

```diff
@@ -336,7 +336,7 @@ int manager_handle_action(
                 }
         }
 
-        /* Locking is handled differently from the rest. */
+        /* Locking and greeter activation is handled differently from the rest. */
```

Comment:
I am relooking at this. I am second guessing changing this comment here, the dbus signal is emitted in logind-button.c for SecureAttentionKey, and not logind-action.c

Reply:
Argh, I can't seem to get the CI to pass now...

Reply:
@poettering Is there anything else I need to change? I have attempted to address all requested changes in the first draft, although GitHub is saying there is a change requested, but I am pretty sure I checked them all off, so I don't know what I am missing